app/controllers/public_pages_controller.rb

```diff
@@ -50,6 +50,16 @@ def page_not_found
 
   def tax_questions; end
 
+  def stimulus
+    # the copy on this page is wrong and until we get it updated, redirect to beginning of intake
+    redirect_to_beginning_of_intake
+  end
+
+  def full_service
+    session[:source] = "full-service"
```

Comment:
The other things I've seen only set the `session[:source]` if it's not already set. Should this stick to that behavior, too?

Reply:
Fascinating. Thanks!